core/types/calls.cc

```diff
@@ -1363,6 +1363,34 @@ class Magic_buildArray : public IntrinsicMethod {
     }
 } Magic_buildArray;
 
+class Magic_buildRange : public IntrinsicMethod {
+public:
+    void apply(const GlobalState &gs, DispatchArgs args, const Type *thisType, DispatchResult &res) const override {
+        ENFORCE(!args.args.empty(), "Magic_buildRange called without argument");
```

Comment:
I think this is reasonable, given that there's no user-visible way to call this with zero args.